lib/src/widgets/html_editor_widget_web.dart

```diff
@@ -47,9 +47,6 @@ class _HtmlEditorWidgetWebState extends State<HtmlEditorWidget> {
   /// The view ID for the IFrameElement. Must be unique.
   late String createdViewId;
 
-  /// The actual height of the editor, used to automatically set the height
-  late double actualHeight;
```

Comment:
IMO, we don't  delete it, we should use it in a way that we can reuse it later. Suggestion: use other property to check.